active.js

```diff
@@ -31,10 +31,17 @@ window.addEventListener('hashchange', () => {
  * Svelte Action for automatically adding the "active" class to elements (links, or any other DOM element) when the current location matches a certain path.
  * 
  * @param {HTMLElement} node - The target node (automatically set by Svelte)
- * @param {string} path - Path expression that makes the link active when matched (must start with '/' or '*')
+ * @param {string} [path] - Path expression that makes the link active when matched (must start with '/' or '*'); default is the link's href
  * @param {string} [className] - CSS class to apply to the element when active; default value is "active"
  */
 export default function active(node, path, className) {
+    if (!path) {
+        path = node.getAttribute('href')
```

Comment:
I'm a bit thorn here: should we check if this is an anchor tag? Or should we just accept a "href" attribute blindly? (I'm leaning towards the second, as it shouldn't really matter, but would like to get your opinion too)

Reply:
Uh, it didn't even occur to me that `use:active` can also be used on other elements. I guess in that case it should only fall back to a `href`attribute. If none exists, it should do as before and complain that no path was given.

I will change this accordingly.